.github/workflows/e2e.maven.workflow_dispatch.main.default.slsa3.yml

```diff
@@ -0,0 +1,49 @@
+name: Maven e2e test - simple
+on:
+  schedule:
+    - cron: "0 6 * * *"
+  workflow_dispatch:
+
+permissions: read-all
+
+env:
+  GH_TOKEN: ${{ github.token }}
+
+jobs:
+  bootstrap:
```

Comment:
As @laurentsimon mentioned have a look at the Node.js builder tests and try to copy that structure.

Concretely, for example, we will need to "bootstrap" to generate a `workflow_dispatch` when the workflow is triggered via a `schedule` event as these are different events with a slightly different event structure.

In this case we want to test the `workflow_dispatch` event so when the event is a `schedule` event, we will call the GitHub API to dispatch the workflow and finish. Then the subsequent `workflow_dispatch` can actually run the test.

Reply:
Looks good! great job creating the `e2e-maven-push.sh`. Sorry our tests are so complicated... 😢